src/mn_menu.c

```diff
@@ -3497,10 +3498,13 @@ void M_Drawer(void)
                 if (patch_lump >= 0)
                 {
                     patch_priority = lumpinfo[patch_lump].handle.priority;
+                    max_priority = MAX(max_priority, patch_priority);
                 }
             }
 
-            if ((patch_lump < 0 || patch_priority < bigfont_priority)
+            if ((patch_lump < 0
+                 || patch_priority < bigfont_priority
+                 || patch_priority < max_priority)
```

Comment:
Fix [Avactor](https://doomwiki.org/wiki/Avactor:_End_of_the_Fifth_Circle) with mbf.wad in autoload:

![woof0004](https://github.com/user-attachments/assets/c1dd8eef-c7a4-4fb1-8ec3-b86233f06762)

Reply:
Ouch!